Libraries/StyleSheet/processTextDecorationLine.android.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule processTextDecorationLine
+ */
+'use strict';
+
+const textDecorationLineBitFlags = {
+  'none': 0,
+  'underline': 1 << 0,
+  'line-through': 1 << 1,
+};
+
+// textDecorationLine can be a string with a single enum value, a string with
+// space-separated enum values, or an array of enum values.  Because ReactAndroid
```

Comment:
An array with the values is the preferred mechanism for specifying multiple text decoration lines.  The space-separated style for multiple values is preserved for compatibility with the current version of react-native, for similarity / familiarity with the CSS text-decoration-line, and because it's cheap and easy to support.